### ISSUE TITLE
Localizations are installed in non-FHS-compliant locations

### ISSUE THREAD

**Issue by mgorny** [blissd/fotema#146]
Fotema currently installs localization files into `${prefix}/i18n` directory. Such a location is not compliant with the [Filesystem Hierarchy Standard](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html) and as such cannot be properly packaged for Linux distributions. Please fix Fotema to use a standard directory, i.e. a subdirectory of `${prefix}/share`.

**Comment by blissd** [comment]
Don't package Fotema for your distribution, please just use the Flatpak.

**Comment by mgorny** [comment]
Please don't do that to us. Users need working packages, Flatpak doesn't work well outside idealized environments.

**Comment by toralf** [comment]
> Don't package Fotema for your distribution, please just use the Flatpak.

Which does not address the reported issue being not compliant with the FHS, right?

**Comment by blissd** [comment]
I'm sorry for coming of as dismissive of your needs, and you are of course free to package and distribute Fotema how you wish.

However, I only target Flatpak, and by extension Flathub, and I don't want to make changes to Fotema that makes doing so awkward for me. Unfortunately for us both I have found i18n to be quite painful with Rust and Linux, with the defacto gettext not really working well. Consequently, I've used Fluent for localization which works well for me... but doesn't integrate well with a number of GTK/GNOME/other linux tools. 

Putting the Fluent localization files in the FHS location worked when testing locally... but something in the Flathub build infrastructure expect gettext 'po' files, I think, and was breaking localization in the builds it produced. 

Long story short, I'm targeting Flatpak/Flathub and not using gettext for localization... so something has to give and that thing is where I put the localisation when packaging.

**Comment by mgorny** [comment]
Could you then use a custom directory, e.g. `share/fotema`? That would be compliant with FHS and not cause issues with tools expecting gettext.

**Comment by blissd** [comment]
I have seen a distribution packaging Fotema that applied a patch to the meson build files to just change the path that the i18n files were deployed to, so I think that will be the easiest option for you. Ssorry, I can't find a link to that right now, but if I do I'll update this issue to include it.

**Comment by blissd** [comment]
> Could you then use a custom directory, e.g. `share/fotema`? That would be compliant with FHS and not cause issues with tools expecting gettext.

Hmm... ok, I'll take a look at doing that.

**Comment by NelvinBHV** [comment]
Just fix the line with i18ndir in meson.build
sed -i "s|i18ndir = prefix / 'i18n'|i18ndir = prefix / 'share' / 'i18n'|g" meson.build
You now able to compile with FHS-compliant locations.

**Comment by NelvinBHV** [comment]
> Please don't do that to us. Gentoo users need working packages, Flatpak doesn't work well outside idealized environments.

Are you working on an ebuild to?

**Comment by mgorny** [comment]
> Are you working on an ebuild too?

1.10.0 is already in `::gentoo`.

**Comment by NelvinBHV** [comment]
> > Are you working on an ebuild too?
> 
> 1.10.0 is already in `::gentoo`.

Thanks, it is brandnew.
